### PR TITLE
DOC-6342 reinstate log flags as aliases

### PR DESCRIPTION
**Pull request by andf-crl**
Addresses: DOC-6342

- Reinstates 8 deprecated log flags as aliases.
	- Reconciled new descriptions (`pkg/cli/cliflags/flags.go`), with also-new descriptions (`pkg/cli/flags_test.go`), with slightly different `cockroach -h` output from the v23.1.0-alpha.8 binary, with the existing docs copy ([live](https://www.cockroachlabs.com/docs/stable/cockroach-start.html#logging)). Let me know if some of the previous copy is no longer correct in the new reality of cockroach log flag aliases.

Suggested

- Some tweaks to your [PR](https://github.com/cockroachdb/cockroach/pull/93255); just minor consistency fixes:
	- Extra trailing `}` character in descriptions to the following flags in file [`pkg/cli/cliflags/flags.go`](https://github.com/cockroachdb/cockroach/pull/93255/files#diff-db7714cb00e3771c8f364fb2424a85323aaf11f5e90ae8437f9e32bc2f3893da):
		- `--log-file-verbosity`
		- `--redactable-logs`
	- Convert `"` to `` ` `` for `Description` to flag `--log-file-max-size` in file [`pkg/cli/cliflags/flags.go`](https://github.com/cockroachdb/cockroach/pull/93255/files#diff-db7714cb00e3771c8f364fb2424a85323aaf11f5e90ae8437f9e32bc2f3893da).

Staging

- [v23.1/cockroach-start.md](https://deploy-preview-16619--cockroachdb-docs.netlify.app/docs/dev/cockroach-start.html#logging)

### CODE REVIEW

**Review by knz — APPROVED** [review]
thank you

**Comment by andf-crl** [comment]
@knz thanks!

**Comment by mdlinville** [comment]
Since the DOC issue is assigned to me, let me quickly review this so we can get it merged.

File: _includes/v23.1/misc/logging-flags.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-`--sql-audit-dir` | **Deprecated.** This is now configured with [`--log`](configure-logs.html#flag) or [`--log-config-file`](configure-logs.html#flag) and a YAML payload. If non-empty, output the `SENSITIVE_ACCESS` [logging channel](logging-overview.html#logging-channels) to this directory.<br><br>Note that enabling `SENSITIVE_ACCESS` logs can negatively impact performance. As a result, we recommend using the `SENSITIVE_ACCESS` channel for security purposes only. For more information, see [Logging use cases](logging-use-cases.html#security-and-audit-monitoring).
+`--log` |  Configure logging parameters by specifying a YAML payload. For details, see [Configure logs](configure-logs.html#flag). If a YAML configuration is not specified, the [default configuration](configure-logs.html#default-logging-configuration) is used.<br/><br/>`--log-config-file` can also be used.<br/><br/>**Note:** The logging flags below cannot be combined with `--log`, and can be defined instead in the YAML payload.
```
Comment (mdlinville) [inline _includes/v23.1/misc/logging-flags.md:3]:
```suggestion
`--log` |  Configure logging parameters by specifying a YAML payload. For details, see [Configure logs](configure-logs.html#flag). If a YAML configuration is not specified, the [default configuration](configure-logs.html#default-logging-configuration) is used.<br/><br/>`--log-config-file` can also be used.<br/><br/>**Note:** The logging flags below cannot be combined with `--log`, but can be defined instead in the YAML payload.
```

File: _includes/v23.1/misc/logging-flags.md [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
+`--log-config-file` |  Configure logging parameters by specifying a path to a YAML file. For details, see [Configure logs](configure-logs.html#flag). If a YAML configuration is not specified, the [default configuration](configure-logs.html#default-logging-configuration) is used.<br/><br/>`--log` can also be used.<br/><br/>**Note:** The logging flags below cannot be combined with `--log-config-file`, and can be defined instead in the YAML file.
```
Comment (mdlinville) [inline _includes/v23.1/misc/logging-flags.md:4]:
```suggestion
`--log-config-file` |  Configure logging parameters by specifying a path to a YAML file. For details, see [Configure logs](configure-logs.html#flag). If a YAML configuration is not specified, the [default configuration](configure-logs.html#default-logging-configuration) is used.<br/><br/>`--log` can also be used.<br/><br/>**Note:** The logging flags below cannot be combined with `--log-config-file`, but can be defined instead in the YAML file.
```

File: _includes/v23.1/misc/logging-flags.md [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
+`--log-dir` | An alias for the [`--log`](configure-logs.html#flag) flag, for configuring the log directory where log files are stored and written to. Specifically, `--log-dir=XXX` is an alias for `--log='file-defaults: {dir: XXX}'`.<br/><br/>Setting `--log-dir` to a blank directory (`--log-dir=""`) disables logging to files.
+`--log-group-max-size` | An alias for the [`--log`](configure-logs.html#flag) flag, for configuring the maximum logging group (i.e., `cockroach`, `cockroach-sql-audit`, `cockroach-auth`, `cockroach-sql-exec`, `cockroach-pebble`) size before deleting the oldest log file. Specifically, `--log-group-max-size=XXX` is an alias for `--log='file-defaults: {max-group-size: XXX}'`. Accepts a valid file size, such as `--log-group-max-size=1GiB`.<br/><br/>**Default:** `100MiB`
```
Comment (mdlinville) [inline _includes/v23.1/misc/logging-flags.md:6]:
```suggestion
`--log-group-max-size` | An alias for the [`--log`](configure-logs.html#flag) flag, for configuring the maximum size for a logging group (for example, `cockroach`, `cockroach-sql-audit`, `cockroach-auth`, `cockroach-sql-exec`, `cockroach-pebble`), after which the oldest log file is deleted. `--log-group-max-size=XXX` is an alias for `--log='file-defaults: {max-group-size: XXX}'`. Accepts a valid file size, such as `--log-group-max-size=1GiB`.<br/><br/>**Default:** `100MiB`
```

File: _includes/v23.1/misc/logging-flags.md [hunk 4]
```diff
@@ -1,11 +1,12 @@
...
+`--log-file-max-size` | An alias for the [`--log`](configure-logs.html#flag) flag, for configuring the maximum size of log files. before a new log file is created. Specifically, `--log-file-max-size=XXX` is an alias for `--log='file-defaults: {max-file-size: XXX}'`. Accepts a valid file size, such as `--log-file-max-size=2MiB`. **Requires** logging to files.<br/><br/>**Default:** `10MiB`
```
Comment (mdlinville) [inline _includes/v23.1/misc/logging-flags.md:7]:
```suggestion
`--log-file-max-size` | An alias for [`--log`](configure-logs.html#flag), used to specify the maximum size that a log file can grow before a new log file is created. `--log-file-max-size=XXX` is an alias for `--log='file-defaults: {max-file-size: XXX}'`. Accepts a valid file size, such as `--log-file-max-size=2MiB`. **Requires** logging to files.<br/><br/>**Default:** `10MiB`
```

File: _includes/v23.1/misc/logging-flags.md [hunk 5]
```diff
@@ -1,11 +1,12 @@
...
+`--log-file-verbosity` | An alias for the [`--log`](configure-logs.html#flag) flag, for configuring log output [severity level](logging.html#logging-levels-severities). Specifically, `--log-file-verbosity=XXX` is an alias for `--log='file-defaults: {filter: XXX}'`. When a severity is specified, such as `--log-file-verbosity=WARNING`, only log messages which are at or above the specified severity level are written to the target log file. **Requires** logging to files.<br/><br/>**Default:** `INFO`
```
Comment (mdlinville) [inline _includes/v23.1/misc/logging-flags.md:8]:
```suggestion
`--log-file-verbosity` | An alias for [`--log`](configure-logs.html#flag), used to specify the minimum [severity level](logging.html#logging-levels-severities) of messages that are logged. `--log-file-verbosity=XXX` is an alias for `--log='file-defaults: {filter: XXX}'`. When a severity is specified, such as `--log-file-verbosity=WARNING`, log messages that are below the specified severity level are not written to the target log file. **Requires** logging to files.<br/><br/>**Default:** `INFO`
```

File: _includes/v23.1/misc/logging-flags.md [hunk 6]
```diff
@@ -1,11 +1,12 @@
...
+`--logtostderr` |  An alias for the [`--log`](configure-logs.html#flag) flag, for configuring log output to the `stderr` sink. Specifically, `--logtostderr=XXX` is an alias for `--log='sinks: {stderr: {filter: XXX}}'`. Accepts a valid [severity level](logging.html#logging-levels-severities). If no value is specified, the default value for the command is inferred as follows: `INFO` for server commands, `WARNING` for client commands.<br/><br/>Setting `--logtostderr=NONE` disables logging to `stderr`.<br><br>**Default:** `UNKNOWN`
```
Comment (mdlinville) [inline _includes/v23.1/misc/logging-flags.md:9]:
```suggestion
`--logtostderr` |  An alias for [`--log`](configure-logs.html#flag), to optionally output log messages at or above the configured  [severity level](logging.html#logging-levels-severities) to the `stderr` sink. `--logtostderr=XXX` is an alias for `--log='sinks: {stderr: {filter: XXX}}'`. Accepts a valid [severity level](logging.html#logging-levels-severities). If no value is specified, by default messages related to server commands are logged to `stderr` at `INFO` severity and above, and messages related to client commands are logged to `stderr` at `WARNING` severity and above.<br/><br/>Setting `--logtostderr=NONE` disables logging to `stderr`.<br><br>**Default:** `UNKNOWN`
```

File: _includes/v23.1/misc/logging-flags.md [hunk 7]
```diff
@@ -1,11 +1,12 @@
...
+`--no-color` | An alias for the [`--log`](configure-logs.html#flag) flag, for configuring colorization of log output to the `stderr` sink. Specifically, `--no-color=XXX` is an alias for `--log='sinks: {stderr: {no-color: XXX}}'`. Accepts either `true` or `false`.<br/><br/>When set to `false`, messages logged to `stderr` are colorized based on [severity level](logging.html#logging-levels-severities).<br><br>**Default:** `false`
```
Comment (mdlinville) [inline _includes/v23.1/misc/logging-flags.md:10]:
```suggestion
`--no-color` | An alias for [`--log`](configure-logs.html#flag) flag, used to control whether log output to the `stderr` sinc is colorized. `--no-color=XXX` is an alias for `--log='sinks: {stderr: {no-color: XXX}}'`. Accepts either `true` or `false`.<br/><br/>When set to `false`, messages logged to `stderr` are colorized based on [severity level](logging.html#logging-levels-severities).<br><br>**Default:** `false`
```

File: _includes/v23.1/misc/logging-flags.md [hunk 8]
```diff
@@ -1,11 +1,12 @@
...
+`--redactable-logs` | An alias for the [`--log`](configure-logs.html#flag) flag, for configuring [redaction markers](configure-logs.html#redact-logs) in log messages. Specifically, `--redactable-logs=XXX` is an alias for `--log='file-defaults: {redactable: XXX}'`. Accepts `true` or `flase`.<br><br>**Default:** `false`
```
Comment (mdlinville) [inline _includes/v23.1/misc/logging-flags.md:11]:
```suggestion
`--redactable-logs` | An alias for [`--log`](configure-logs.html#flag) flag, used to whether [redaction markers](configure-logs.html#redact-logs) are used in place of secret or sensitive information in log messages. `--redactable-logs=XXX` is an alias for `--log='file-defaults: {redactable: XXX}'`. Accepts `true` or `false`.<br><br>**Default:** `false`
```

File: _includes/v23.1/misc/logging-flags.md [hunk 9]
```diff
@@ -1,11 +1,12 @@
...
+`--sql-audit-dir` | An alias for the [`--log`](configure-logs.html#flag) flag, for configuring log output of the `SENSITIVE_ACCESS` [logging channel](logging-overview.html#logging-channels) to the indicated directory. Specifically, `--sql-audit-dir=XXX` is an alias for `--log='sinks: {file-groups: {sql-audit: {channels: SENSITIVE_ACCESS, dir: ...}}}'`.<br><br>Note that enabling `SENSITIVE_ACCESS` logs can negatively impact performance. As a result, we recommend using the `SENSITIVE_ACCESS` channel for security purposes only. For more information, see [Logging use cases](logging-use-cases.html#security-and-audit-monitoring).
```
Comment (mdlinville) [inline _includes/v23.1/misc/logging-flags.md:12]:
```suggestion
`--sql-audit-dir` | An alias for [`--log`](configure-logs.html#flag), used to optionally confine log output of the `SENSITIVE_ACCESS` [logging channel](logging-overview.html#logging-channels) to a separate directory. `--sql-audit-dir=XXX` is an alias for `--log='sinks: {file-groups: {sql-audit: {channels: SENSITIVE_ACCESS, dir: ...}}}'`.<br><br>Enabling `SENSITIVE_ACCESS` logs can negatively impact performance. As a result, we recommend using the `SENSITIVE_ACCESS` channel for security purposes only. For more information, refer to [Security and Audit Monitoring](logging-use-cases.html#security-and-audit-monitoring).
```

**Review by mdlinville — CHANGES_REQUESTED** [review]
Here are some small copyedit suggestions. Thanks for the change. By the way, I could not figure out what actually changed in lines 3 and 4 before my suggested edit, even after careful scrutiny.

**Review by mdlinville — APPROVED** [review]

**Pull request merged**